### DISCUSSION TITLE
Sesión 21 de marzo

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#479] [Q&A] [answered]
Buenas tardes, ¿Saben si abra sesión?

**Comment by BattiestWinter** [comment, marked as answer]
La verdad no sé, el mc comentó que tal vez no, pero tampoco lo confirmó. A mi por lo menos no me aparece aún el canal de la práctica 6, a lo mejor no hay live

**Reply by GloriaCoralCerecedo** [reply]
Gracias
